### PR TITLE
Add demos

### PR DESCRIPTION
**Pull request by maxammann**
Add links to latest demos

### CODE REVIEW

**Comment by maxammann** [comment]
@nyurik which of the demos work for you?

**Comment by nyurik** [comment]
Only the first demo works. The second one domain is dead. The 3rd tells me i don't have a gpu

**Comment by nyurik** [comment]
P.S. First was on chrome.  I also tried it in FF on linux - same result

File: README.md [hunk 1]
```diff
@@ -76,6 +76,12 @@ graphics are achievable using the current stack.
 
 ([Older Talk on YouTube](https://www.youtube.com/watch?v=KFk8bOtJzCM))
 
+## Demos
+
+- [WebGL](https://webgl.demo.maplibre-rs.maplibre.org)
+- [WebGL with multithreading](https://webgl-multithreading.demo.maplibre-rs.maplibre.org) - Does not work on Safari right now
```
Comment (maxammann) [inline README.md:82]:
```suggestion
- [WebGL with multithreading](https://webgl-multithreaded.demo.maplibre-rs.maplibre.org) - Does not work on Safari right now
```

**Comment by maxammann** [comment]
Upsi, wrong domain. Fixed.

**Pull request merged**